### ISSUE TITLE
Nuget pack command does not include all files

### ISSUE THREAD

**Issue by elv1s42** [NuGet/Home#3678]
Hello.

I've created nuspec file for my C# project.

```
<?xml version="1.0" encoding="utf-8"?>
<package >
  <metadata>
    <id>$id$</id>
    <version>$version$</version>
    <title>$title$</title>
    <authors>$author$</authors>
    <owners>$author$</owners>
    <licenseUrl>https://github.com/GHPReporter/Ghpr.Core/blob/master/LICENSE</licenseUrl>
    <projectUrl>https://github.com/GHPReporter/Ghpr.Core</projectUrl>
    <iconUrl>https://github.com/GHPReporter/GHPReporter.github.io/blob/master/img/logo-small.png?raw=true</iconUrl>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <description>Easy-to-use test run reporter</description>
    <releaseNotes></releaseNotes>
    <copyright>Copyright Evgeniy Kosjakov © 2016</copyright>
    <tags>.NET package HTML report reporting tool</tags>
  </metadata>
  <files>
    <file src=".\bin\Release\*.dll" target="lib" />
    <file src=".\bin\Release\*.config" target="lib" />
  </files>
</package>
```

I want to iclude all .dll failes and one .config file.
When I am opening this .nuspec file with NuGet Package explorer everything is fine:
![image](https://cloud.githubusercontent.com/assets/6689884/19475617/b9da93cc-953d-11e6-9356-c53d2451db16.png)
I can see all files inside 'package contents'.
But when I run `pack` command, there are no such files inside .nupkg file:
![image](https://cloud.githubusercontent.com/assets/6689884/19475673/faab3064-953d-11e6-80bf-58cff1e9b10b.png)
I can see only one .dll file.

What am I doing wrong?
How can I add all files to my .nupkg?

Here is my NuGet.exe pack command:
![image](https://cloud.githubusercontent.com/assets/6689884/19475725/3b042d64-953e-11e6-84dc-8b66c499b3ec.png)

Thanks!

**Comment by yishaigalatzer** [comment]
I believe that because you are packing the csproj and not the nuspec the files section is ignored and the nuspec is used only for metadata.

Also note that you do not want to pack newtonsoft.json inside your package, you should depend on newtonsoft.json package instead.

@rohit21agrawal can you provide with guidance on how to include extra files from the output and @karann-msft can you point to appropriate docs?

**Comment by yishaigalatzer** [comment]
Talking to @rohit21agrawal it seems like we fixed this issue, he is going to jump here and provide the build where this starts working as you expect.

**Comment by elv1s42** [comment]
Thanks @yishaigalatzer !
The main thing that I need is to able to create NuGet package with `.dll.config` file.
In my case, I just need one dll (Ghpr.Core.dll) and one config file (Ghpr.Core.dll.config). 
Newtonsoft.Json is a dependency, and it installs correctly.

When I am trying to pack `.csproj` file I expect `.config` file to be included into `.nupkg`

**Comment by rohit21agrawal** [comment]
@elv1s42 please try out the latest 3.6.0-rc nuget.commandline build from : https://dotnet.myget.org/F/nuget-build/api/v2/package/NuGet.CommandLine

you will find nuget.exe under the tools folder of the nupkg.

**Comment by elv1s42** [comment]
Hi @rohit21agrawal 

I tried new nuget.exe, and it works as expected:
![image](https://cloud.githubusercontent.com/assets/6689884/19553715/ff49ca5a-96bd-11e6-945d-ba576e593839.png)

Putting some more info for everyone who will need to do the same.
I used this .nuspec file:

```
<?xml version="1.0" encoding="utf-8"?>
<package >
  <metadata>
   ...
  </metadata>
  <files>
    <file src=".\bin\Release\*.config" target="lib\net452" />
  </files>
</package>
```

This is a screenshot of cmd run:
![image](https://cloud.githubusercontent.com/assets/6689884/19553878/cc23f9ec-96be-11e6-976c-3226a6d9a848.png)

So I guess this new version works fine. The only problem that I have for now is to create a package using appveyor CI.
Thank you, I suppose we can close this issue as it is already fixed in the latest 3.6.0-rc nuget.commandline build.
